entities/SwordEntity.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using JetBrains.Annotations;
+using enums;
+using UnityEngine;
+
+namespace TotemEntities
+{
+    [Serializable]
+    public class TotemSword
+    {
```

Comment:
TotemSword should implemet the ITotemAsset iterface so it can be used with Totem services